lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -111,8 +111,14 @@ struct nvme_probe_ctx {
 	const char *names[NVME_MAX_CONTROLLERS];
 };
 
+enum timeout_action {
+	TIMEOUT_ACTION_NONE = 0,
+	TIMEOUT_ACTION_RESET,
+	TIMEOUT_ACTION_ABORT
```

Comment:
It would be good to keep a trailing comma here to reduce the diff if we ever add another entry.